website/src/components/mdx/auto-linked-heading.tsx

```diff
@@ -0,0 +1,59 @@
+import { Link } from "gatsby";
+import React, { FC } from "react";
```

Comment:
Would prefer `FunctionComponent` instead of `FC`. Consistency :-)

Reply:
Changed. In future work I would like to use `FC`. That is less noisy and does not occupy almost the full print-width when component properties are also specified.